### PR TITLE
fixes in the PoC first notebook

### PR DESCRIPTION
**Pull request by rnyak**
This PR 

- fixing the correct raw dataframe to be fed to both models so the user and item catalogs wont be confused
- training retrieval model first and then the ranking (logical order)

### CODE REVIEW

File: examples/Building-and-deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb [hunk 1]
```diff
@@ -115,43 +115,15 @@
    "source": [
```
Comment (sararb) [inline examples/Building-and-deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb:115]: <p>Maybe we could add the section <code>Exporting query (user) model</code> here to keep the logical flow. We can add a sentence like <code>We export the query tower to use it later during the deployment&nbsp;stage.</code>  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

**Comment by karlhigley** [comment]
rerun tests

**Comment by bschifferer** [comment]
rerun tests

**Comment by rnyak** [comment]
rerun tests

**Review by sararb — APPROVED** [review]

**Pull request merged**